### PR TITLE
privnet show: show DHCP informations in the command output

### PR DESCRIPTION
**Pull request by mcorbin**
```
exo privnet show da9b7b1b-8906-4e0d-9f42-09a985044f02

┼──────────┼──────┼─────────────────────────┼─────────────────────────────────────────┼──────────────────────────────────────┼
│   ZONE   │ NAME │          DHCP           │             VIRTUAL MACHINE             │          VIRTUAL MACHINE ID          │
┼──────────┼──────┼─────────────────────────┼─────────────────────────────────────────┼──────────────────────────────────────┼
│ ch-gva-2 │ foo  │ 11.0.0.1-11.0.0.200 /24 │ bar                                     │ afdff495-1be3-4977-bf60-2ec980568a3d │
│          │      │                         │ VM-f466061e-a584-4a90-bb8c-88e3fe4b4097 │ f466061e-a584-4a90-bb8c-88e3fe4b4097 │
┼──────────┼──────┼─────────────────────────┼─────────────────────────────────────────┼──────────────────────────────────────┼
```

### CODE REVIEW

**Comment by mcorbin** [comment]
I don't know which view will be better for this command.

> IMHO, the missing key information here is the IP of the machine. Does it have a static lease, does it have a dynamic one.

I can probably update the command to show the static leases, but not the dynamic leases.

**Comment by greut** [comment]
We can imagine what the portal will look like.

```
Network:          <name>
Description:      <display_name>
Zone:             CH-GVA-2
IP Range:         10.0.0.1 - 10.0.0.120 / 25
# Instances:      10
```

| ID | name | Public IP | Internal IP |
|----|------|-----|----|
| ... | my-vm | 159.... | 10.0.0.10 |
| .... | a-vm | 159.... | auto |

The list of VMs can be within the show.

**Comment by mcorbin** [comment]
New output:

```
exo privnet show da9b7b1b-8906-4e0d-9f42-09a985044f02
Network:     foo
Description: test
Zone:        ch-gva-2
IP Range:    11.0.0.1-11.0.0.200 /24
# Instances: 1
┼──────────────────────────────────────┼─────────────────────────────────────────┼─────────────────┼─────────────┼
│                  ID                  │                  NAME                   │    PUBLIC IP    │ INTERNAL IP │
┼──────────────────────────────────────┼─────────────────────────────────────────┼─────────────────┼─────────────┼
│ 3c2b2ec6-c7c4-41da-a02f-736a67a93dbd │ VM-3c2b2ec6-c7c4-41da-a02f-736a67a93dbd │ 159.100.241.218 │ 11.0.0.5    │
┼──────────────────────────────────────┼─────────────────────────────────────────┼─────────────────┼─────────────┼
```

The Internal IP is replaced by "n/a" if it doesn't exists.

**Comment by greut** [comment]
@mcorbin cool, good job!

File: cmd/exo/cmd/privnet_show.go [hunk 1]
```diff
@@ -27,21 +29,32 @@ var privnetShowCmd = &cobra.Command{
...
-		name := network.Name
+		table.SetHeader([]string{"ID", "Name", "Public IP", "Internal IP"})
+
 		if len(vms) > 0 {
 			for _, vm := range vms {
-				table.Append([]string{zone, name, vm.Name, vm.ID.String()})
-				zone = ""
-				name = ""
+
+				privateNic := vm.NicByNetworkID(*network.ID)
```
Comment (greut) [inline cmd/exo/cmd/privnet_show.go]: this is already done by `privnetDetails`, feel free to enrich it.

Comment (greut) [reply cmd/exo/cmd/privnet_show.go]: fair enough.

**Pull request merged**